### ISSUE TITLE
[FEATURE]: Persistent search type

### ISSUE THREAD

**Issue by iSentrie** [citizenfx/txAdmin#1010]
### Scope

Web

### Feature Description

I would like to request a feature which would keep the search type selected and wouldn't reset after reloading a page.

### Use Case

I usually search by identifiers and every time I jump between pages I have to select "Player ID's" search type.

### Proposed Solution

Make selected search type persistent and that it wouldn't reset each time when we surf pages.

### Additional Info

![image](https://github.com/user-attachments/assets/c1a0a8e1-e685-4a5a-b61d-4bc9ac9a6561)

**Comment by tabarra** [comment]
I agree that this is a need, but I don't know how to do it properly.  
I do not want to change the default to be persistent, as a lot of people will get confused or ask the opposite, so this needs to be an intentional change, I just don't know how to do the UI for that properly.  
  
**What do you think of this?**  
  
![image](https://github.com/user-attachments/assets/036421f9-1109-4767-a597-645c89ada7a9)

**Comment by iSentrie** [comment]
> I agree that this is a need, but I don't know how to do it properly. I do not want to change the default to be persistent, as a lot of people will get confused or ask the opposite, so this needs to be an intentional change, I just don't know how to do the UI for that properly.
> 
> **What do you think of this?**
> 
> ![image](https://private-user-images.githubusercontent.com/1808295/386996037-036421f9-1109-4767-a597-645c89ada7a9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4NzU4NzYsIm5iZiI6MTczMTg3NTU3NiwicGF0aCI6Ii8xODA4Mjk1LzM4Njk5NjAzNy0wMzY0MjFmOS0xMTA5LTQ3NjctYTU5Ny02NDVjODlhZGE3YTkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTExNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMTdUMjAzMjU2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MWE4ZWYwYjU2ZGI3MjQ2NWQ4YTEyYWQxM2E5NzIzMWUzYTk0YzY3MjEyNjU5OTMzNmEzOTQyYjE4NGRjMDVlMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.DaY7H0T9v7zkFo3NQKDZhaOgoQXWhFC-aSFDnx7iZEs)

Perfect 👌

**Comment by tabarra** [comment]
Oh yeah, forgot to close this.  
This is available as of v8.0.
